test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -91,11 +88,13 @@ class StructuredFormDataBuilderSpec extends Spec {
         "2_field" -> "fieldValue2"
       ),
       objectStructure(
-        "field" -> arrayNode(
-          textNode("fieldValue1"),
-          textNode("fieldValue2")
-        )
-      )
+        field(
+          "field",
+          arrayNode(
+            textNode("fieldValue1"),
+            textNode("fieldValue2")
+          ),
+          Map.empty))
```

Comment:
Make our lives easy & remove the duplication by giving the alternateFieldNames parameter of field(...) a default, Map.empty.